### DISCUSSION TITLE
The final result can be paginated

### DISCUSSION THREAD

**Discussion by nappalm** [uwdata/arquero#385] [Q&A]
How could I use the limit and offset to paginate my result, since it is too much information to process. Since I need the total count of records and the paginated result.

Any recommendations?

```
const result = from(data)
    .join_left(....)
    .join_left(....)
    .filter((row) => row.xyz=== undefined)
    .reify()
```

**Comment by jheer** [comment]
Is the issue that the table produced by the joins is too large to hold in memory?

One possibility is to break up the original `data` into subsets, and then join and process those individually. You could construct the total count as the sum of row counts for each processed subset. Of course, getting this right will depend on how much the joins increase the row count.
